packages/tools/devtools/devtools-browser-extension/package.json

```diff
@@ -17,18 +17,20 @@
 		"build:compile": "fluid-build . --task compile",
 		"build:test": "tsc --project ./e2e-tests/tsconfig.json",
 		"build:webpack": "npm run webpack",
+		"check:biome": "biome check .",
+		"check:prettier": "prettier --check . --cache --ignore-path ../../../../.prettierignore",
 		"clean": "rimraf --glob coverage dist nyc \"**/*.tsbuildinfo\" \"**/*.build.log\"",
 		"eslint": "eslint src",
 		"eslint:fix": "eslint src --fix",
-		"format": "npm run prettier:fix",
+		"format": "fluid-build --task format .",
 		"format-and-build": "npm run format && npm run build",
 		"format-and-compile": "npm run format && npm run build:compile",
+		"format:biome": "biome check --apply .",
+		"format:prettier": "prettier --write . --cache --ignore-path ../../../../.prettierignore",
 		"good-fences": "gf",
-		"lint": "npm run prettier && npm run good-fences && npm run eslint",
-		"lint:fix": "npm run prettier:fix && npm run eslint:fix",
+		"lint": "fluid-build . --task lint",
+		"lint:fix": "fluid-build . --task eslint:fix --task format",
```

Comment:
Curious, there is no explicit `eslint:fix` task defined in the fluid build config file but it's being used all over the repo. Did we default to looking for npm scripts if a task doesn't exist?

Reply:
> Did we default to looking for npm scripts if a task doesn't exist?

As I recall it's always behaved this way for leaf tasks with no dependencies. I tested removing the task definitions that are just `[]` in the fluid build config and those tasks continued to work. So the empty definitions seem superfluous - but they _might_ be used when a project-specific task inherits the parent definition with `"..."` That is, you might need a definition in the config if you want to override/extend that task in an individual project. Haven't looked at the code to check though.

cc: @jason-ha since I told you something different (and wrong I think) about tasks needing to be defined.